src/client/components/AllList.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react"
+import {useEffect, useState} from "react"
```

Comment:
wait, do you need both lines really? I'd expect the wildcart to be a wildcart, no?

Reply:
typescript is not as you expect :p